### ISSUE TITLE
Broken from Polymer 2.4.0...

### ISSUE THREAD

**Issue by sks** [PredixDev/px-context-browser#98]
# Submit an Issue

Hi,

## Expected behavior and actual behavior:

The Context browser should work as expected
## Steps to reproduce the problem:

[Codepen](https://codepen.io/sabithksme/pen/WMoMrZ?&editors=1001)
```
webcomponentsjs+v1.0.22
polymer+v2.4.1

<base href="https://polygit.org/px-*+PredixDev+*/webcomponentsjs+v1.0.22/polymer+v2.4.1/components/">
```

## Environment (_component version number, Browser (including version), operating system, hardware, etc_):

[Codepen](https://codepen.io/sabithksme/pen/WMoMrZ?&editors=1001)

## Screenshots (_optional, but very helpful_):

Error Trace
```
px-app-asset-behavior-graph.es6.js:386 Uncaught Error: An array of route strings is required.
    at AssetGraph.getNodeAtRoute (px-app-asset-behavior-graph.es6.js:386)
    at HTMLElement._activeRouteChanged (px-app-asset-behavior-activatable.html:271)
    at Object.runMethodEffect [as fn] (property-effects.html:813)
    at runEffectsForProperty (property-effects.html:159)
    at runEffects (property-effects.html:125)
    at HTMLElement._propertiesChanged (property-effects.html:1703)
    at HTMLElement._flushProperties (properties-changed.html:325)
    at HTMLElement._flushProperties (property-effects.html:1551)
    at HTMLElement.ready (property-effects.html:1656)
    at HTMLElement.ready (element-mixin.html:551)
```

## Code examples
[Codepen](https://codepen.io/sabithksme/pen/WMoMrZ?&editors=1001)

**Comment by mdwragg** [comment]
Weird, it's working fine on predix-ui.com (which is running Polymer 2.5.0 currently), if you update to 2.5.0, any difference?

@talimarcus @davidrleonard - any ideas?

**Comment by talimarcus** [comment]
I think @davidrleonard is looking into this

**Comment by mdwragg** [comment]
Sweet... Thanks.

**Comment by mdwragg** [comment]
@sks did you see my comment?

**Comment by sks** [comment]
Yep @mdwragg ,

I tried using `2.5.0` in [this example](https://codepen.io/sabithksme/pen/QQdaYG?&editors=1001)

Still the issue persisted.

**Comment by davidrleonard** [comment]
Fixed in [px-app-helpers v2.1.7](https://github.com/PredixDev/px-app-helpers/releases/tag/v2.1.7). The fix is in https://github.com/PredixDev/px-app-helpers/commit/135754a02f69c61e94268d7ef8c35ad576221864 if you're interested.

Re-run bower install and ensure you pick up px-app-helpers at v2.1.7 or later and you shouldn't see this issue any longer.
